### PR TITLE
Update code owners for frontend

### PR DESCRIPTION
**Pull request by natke**
**Description**: Update the codeowners files so appropriate people are assigned for review

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,13 @@
 * @Microsoft/onnxruntime
 
 # Python frontend owners
-orttraining/*.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
```
Comment (nkreeger) [inline CODEOWNERS:4]: Might drop @mrry as well.

**Review by thiagocrepaldi — COMMENTED** [review]
@natke do you know whether Github supports groups as opposed to individuals?

File: CODEOWNERS (hunk 1 above)
Comment (natke) [reply CODEOWNERS:4]: Good point!

**Comment by natke** [comment]
> @natke do you know whether Github supports groups as opposed to individuals?

I don't know that. It would be easier to maintain, wouldn't it?

File: CODEOWNERS (hunk 1 above)
Comment (natke) [reply CODEOWNERS:4]: @nkreeger, would you want to be added as a code owner?

**Comment by SherlockNoMad** [comment]
duplicate of https://github.com/microsoft/onnxruntime/pull/8335 
will close this PR.

**Pull request closed without merging**